client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-application-password-authorization/index.tsx

```diff
@@ -55,9 +60,10 @@ const SiteMigrationApplicationPasswordsAuthorization: Step< {
 
 	useEffect( () => {
 		if ( isStoreApplicationPasswordSuccess ) {
+			dispatch( resetSite( siteId ) );
```

Comment:
Can we make the call conditional on the value of `siteId`? Or at least require the value to be non-zero.
```suggestion
			siteId && dispatch( resetSite( siteId ) );
```